cis_policies_generator/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "cis_policies_generator",
+  "version": "1.0.0",
+  "description": "",
+  "main": "src/index.ts",
+  "scripts": {
+    "start": "set TS_NODE_FILES=true & node -r ts-node/register/transpile-only src/index.ts",
+    "nodemon": "nodemon --watch src/**/*.ts --watch src/**/*.js --exec node -r ts-node/register/transpile-only src/index.ts",
```

Comment:
`nodemon` should probably be a dev dependency so a single `npm install` would get all deps